job_executor/app.py

```diff
@@ -39,6 +36,42 @@ def logger_thread(logging_queue: Queue):
         logger.handle(record)
 
 
+def fix_interrupted_jobs():
+    logger.info('Querying for interrupted jobs')
+    in_progress_jobs = job_service.get_jobs(ignore_completed=True)
```

Comment:
all_jobs instead of in_progress_jobs, maybe?
I assume job_service.get_jobs returns all jobs.